### DISCUSSION TITLE
How to use vehicle per kilometer cost in CVRP?

### DISCUSSION THREAD

**Discussion by atulsinha-zepto** [google/or-tools#3546] [Routing (and legacy CP) questions] [answered]
I understand that the dimensions can be used for arc weights but in order to minimize the overall cost of the problem, I need to optimise on vehicle costs. Different vehicles can have different cost per kilometer.

**Comment by lperron** [comment, marked as answer]
https://google.github.io/or-tools/python/ortools/constraint_solver/pywrapcp.html#RoutingModel.SetArcCostEvaluatorOfVehicle
